src/pages/LoginPage/LoginPage.tsx

```diff
@@ -0,0 +1,306 @@
+      },
+    }}
+  />
+);
+
+const PrimaryButton: React.FC<PrimaryButtonFallbackProps> = (props) => (
+  <Button
+    variant="contained"
+    onClick={props.onClick}
+    disabled={props.disabled || props.loading}
+    sx={props.sx}
+  >
+    {props.loading ? '...' : props.children}
+  </Button>
+);
+
+const EMAIL_REGEX = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
+const PASSWORD_MIN_LENGTH = 6;
+
+export const LoginPage: React.FC<LoginPageProps> = ({
+  loading = false,
+  buttonText = 'Login',
+  disabled = false,
+  onSubmit = () => undefined,
+  forgotPasswordText = 'Forget Password ?',
+  signUpText = 'Sign Up',
+  orYouCanText = 'Or you can Signup with',
+  showSocialIcons = true,
+  showLegalText = true,
+  googleLoginUrl = 'https://accounts.google.com/signin',
+  githubLoginUrl = 'https://github.com/login',
+  facebookLoginUrl = 'https://www.facebook.com/login',
+  privacyPolicyUrl = 'https://policies.google.com/privacy',
+  termsOfServiceUrl = 'https://policies.google.com/terms',
+}) => {
+  const [emailOrUsername, setEmailOrUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  const [errors, setErrors] = useState<{ emailOrUsername: string; password: string }>({
+    emailOrUsername: '',
+    password: '',
+  });
+
+  const theme = useTheme();
+
+  const handleImgError = (e: React.SyntheticEvent<HTMLImageElement>) => {
+    const img = e.currentTarget;
+    const alt = img.alt;
+    if (alt === 'Google' && img.src !== GoogleIcon) {
+      img.src = GoogleIcon ?? '';
+    } else if (alt === 'GitHub' && img.src !== GithubIcon) {
+      img.src = GithubIcon ?? '';
+    } else if (alt === 'Facebook' && img.src !== FacebookIcon) {
+      img.src = FacebookIcon ?? '';
+    }
+  };
+
+  const handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault();
+
+    const newErrors = { emailOrUsername: '', password: '' };
+
+    if (!emailOrUsername) {
+      newErrors.emailOrUsername = 'Email or Username is required.';
+    } else if (emailOrUsername.includes('@') && !EMAIL_REGEX.test(emailOrUsername)) {
+      newErrors.emailOrUsername = 'Please enter a valid email address.';
+    }
+
+    if (!password) {
+      newErrors.password = 'Password is required.';
+    } else if (password.length < PASSWORD_MIN_LENGTH) {
+      newErrors.password = `Password must be at least ${PASSWORD_MIN_LENGTH} characters long.`;
+    }
+
+    if (newErrors.emailOrUsername || newErrors.password) {
+      setErrors(newErrors);
+      return;
+    }
+
+    setErrors(newErrors);
+    onSubmit({ emailOrUsername, password });
+  };
+
+  const isDisabled = disabled || loading || !!errors.emailOrUsername || !!errors.password;
+
+  return (
+    <Box sx={getPageContainerStyles()}>
+      <Box component="form" onSubmit={handleSubmit} sx={getFormStyles(theme)}>
+        <Box sx={getLogoStyles(theme)}>
+          <Logo orientation="horizontal" width="100%" height="100%" alt="Logo" />
+        </Box>
+
+        <FormInput
```

Comment:
FormInput fallback usage lacks an explicit accessible label (no name/id/aria-label). Relying only on placeholder text is not acceptable for accessibility (placeholders can disappear and are not a label substitute).

**Sugerencia:**
```typescript
Ensure FormInput accepts and renders a label element (with htmlFor) or an aria-label/name prop. For example, pass id and label props and render <label htmlFor={id}>Label</label> and <input id={id} name={name} ...>. Also forward and set aria-invalid when errors are present.
```